UnityProject/Assets/Scripts/Items/Devices/AccessRemote.cs

```diff
@@ -0,0 +1,135 @@
+using System;
+using Doors;
+using Doors.Modules;
+using Mirror;
+using UnityEngine;
+
+
+namespace Items.Devices
+{
+	/// <summary>
+	/// An item that controls department doors remotely.
+	/// </summary>
+	public class AccessRemote : NetworkBehaviour, IInteractable<HandActivate>, ICheckedInteractable<HandApply>
```

Comment:
ICheckedInteractable with the handActivate too so it is validated

Reply:
why do you need to validate handactive? We don't need to check for where it's position in the world or anything like that

Reply:
we check to make sure its in their hand